packages/yoastseo/src/languageProcessing/helpers/sentence/memoizedSentenceTokenizer.js

```diff
@@ -0,0 +1,28 @@
+import SentenceTokenizer from "./SentenceTokenizer";
+import { memoize } from "lodash-es";
+
+/**
+ * Returns the sentences from a certain block.
+ *
+ * @param {string} block 					The HTML inside a HTML block.
+ * @param {boolean} [trimSentences=true] 	Whether to trim whitespace from the beginning and end of the sentences or not.
+ *
+ * @returns {Array<string>} The list of sentences in the block.
+ */
+function getSentenceTokenizer( block, trimSentences = true ) {
```

Comment:
Good idea, I changed it to `text` since it technically could be any string of text, it doesn't have to correspond to an HTML block. And in the HTML parser, we only pass the `innerText`, not the whole HTML block.